content/installation/_index.md

```diff
@@ -7,10 +7,11 @@ weight: 20
 
 # Getting Started
 
-## Prerequisites 
+## Prerequisites
 
 To install [superwerker], you need to fulfil the following two prerequisites:
 
+1. A deployment region in which AWS Control Tower is available. superwerker relies on AWS Control Tower which is not available in every AWS region. See details [here](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/)
```

Comment:
Would it make sense to list the currently supported regions here? The AWS region table is really cumbersome ...

Reply:
I can list the regions, but I assume this will change quite often. The workshop will then have out of date information that will need to be corrected. The table is cumbersome but (hopefully) consistently uptodate.

Reply:
Put the AWS region info in installation instruction step 2.